github3/repos/branch.py

```diff
@@ -372,6 +395,93 @@ def users(self, number=-1):
         from .. import users
         return self._iter(int(number), self.users_url, users.ShortUser)
 
+    @decorators.requires_auth
+    def add_teams(self, teams):
+        """
+        Add teams to the protected branch.
+        See https://git.io/vxrfF
+        :param teams:
+        :return:
+        """
+        resp = self._post(self.teams_url, data=teams,
```

Comment:
What happens if I pass in `teams=None` or `teams=[]`?

What should teams be? A list of strings? A list of team ids? This is why we try to document the parameter and add guards to methods.